internal/components/datasource/datasource.cue

```diff
@@ -26,11 +20,9 @@ seqs: [
                 basicAuthPassword: string
                 withCredentials: bool | *false
                 isDefault: bool | *false
-                jsonData?: [string]: _
+                jsonData: string
```

Comment:
This was a hackety-hack to make things a bit simpler to work with. We can change it back - I just didn't feel like spending extra time fiddling with proper typing here, since working with JSON / maps is a bit trickier.

If this is a blocker, I can spend extra time and make it work, otherwise I suggest we leave this out for now and have a separate issue / PR to figure out proper datasource schema.

Let me know what you think about it.

Reply:
ack, I also had difficulty working with maps.